packages/plugin-e2e/tests/as-admin-user/datasource/query-editor/queryEditor.integration.spec.ts

```diff
@@ -33,3 +33,5 @@ test('should be possible to load and execute an existing valid query', async ({
   await expect(panelEditPage.refreshPanel()).toBeOK();
   await expect(panelEditPage.panel.getErrorIcon()).not.toBeVisible();
 });
+
+// DEBUG=pw:browser
```

Comment:
is this annotation needed?

Reply:
Nope, not needed. Thanks for spotting this.